data/contract_docs/contract.md

```diff
@@ -14,6 +14,21 @@ There are logical **_kinds_** (like Time Series Data, Numeric, Histogram, etc),
 
 A **_data type_** definition or declaration in this framework includes both a kind and format. For example, "TimeSeriesWide" is: kind: "Time Series", format: "Wide".
 
+* [Time series](./timeseries.md)
+     * [Wide](./timeseries.md#time-series-wide-format-timeserieswide)
+     * [Long](./timeseries.md#time-series-long-format-timeserieslong-sql-like)
+     * [Multi](./timeseries.md#time-series-multi-format-timeseriesmulti)
+* [Numeric](./numeric.md)
+     * [Wide](./numeric.md#numeric-wide-format-numericwide)
+     * [Many](./numeric.md#numeric-many-format-numericmany)
```

Comment:
This was changed to numeric multi in recent PR https://github.com/grafana/grafana-plugin-sdk-go/pull/551/files

Reply:
LIke this?
```
     * [Multi](./timeseries.md#time-series-multi-format-timeseriesmulti)
* [Numeric](./numeric.md)
     * [Wide](./numeric.md#numeric-wide-format-numericwide)
     * [Multi](./numeric.md#numeric-many-format-numericmany)
```